utils/mp-server/table_definitions.sql

```diff
@@ -89,6 +89,7 @@ CREATE INDEX START_TIME_IDX ON game_info(START_TIME);
 -- CLIENT_VERSION: the version of the wesnoth client used to connect
 -- CLIENT_SOURCE: where the wesnoth client was downloaded from - SourceForge, Steam, etc
 -- USER_NAME: the username logged in with
+-- LEADERS: the leader(s) set or chosen during MP staging. if multiple leaders are found, then they are comma delimited.
```

Comment:
So here it references staging but that is not it since otherwise it would be a single leader, no? I would also mention here that this is an approximation. Unlike all the other data so far we already know this is not accurate in all possible cases. Though of course we hope that in the great majority it is.

Reply:
Updated.